packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -46,7 +46,7 @@ const webpackMerge = require('webpack-merge');
 // It would really help if it happens during architect.validateBuilderOptions, or similar.
 export interface NormalizedBrowserBuilderSchema extends BrowserBuilderSchema {
   assets: AssetPatternObject[];
-  fileReplacements: CurrentFileReplacement[];
+  fileReplacements: FileReplacement[];
```

Comment:
This change is not correct. Post normalization the type is `CurrentFileReplacement` instead of `FileReplacement` (which is `DeprecatedFileReplacement | CurrentFileReplacement`).